tests/app/celery/test_tasks.py

```diff
@@ -1052,3 +1052,14 @@ def test_update_job_to_sent_to_dvla(sample_letter_template, sample_letter_job):
     assert [(n.status == 'sending', n.sent_by == 'dvla') for n in updated_notifications]
 
     assert 'sent to dvla' == Job.query.filter_by(id=sample_letter_job.id).one().job_status
+
+
+def test_update_dvla_job_to_error(sample_letter_template, sample_letter_job):
+    create_notification(template=sample_letter_template, job=sample_letter_job)
+    create_notification(template=sample_letter_template, job=sample_letter_job)
+    update_dvla_job_to_error(job_id=sample_letter_job.id)
+
+    updated_notifications = Notification.query.all()
+    assert [(n.status == 'created', n.sent_by == 'dvla') for n in updated_notifications]
```

Comment:
this only asserts that the array is truthy - ie, that it has a non-zero length (and asserting tuples also just checks that they have contents in 

you'll have to unpack this i'm afraid